README.md

```diff
@@ -2,14 +2,26 @@
 A fork of ResponseCachingMiddleware which provides more flexibility. Note that it uses official aspnetcore code directly (via git submodules and csproj linking).
 
 ## DistributedResponseCache
-For usage see [ResponseCachingMiddleware](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/middleware?view=aspnetcore-3.1), just replace AddResponseCaching with AddDistributedResponseCaching, and UseResponseCaching with UseDistributedResponseCaching.<br>
+Usage is similar to [ResponseCachingMiddleware](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/middleware?view=aspnetcore-3.1). Set the ResponeCachingStrategy to ResponseCachingStrategy.Distributed when calling AddResponseCaching and replace UseResponseCaching with UseCustomResponseCaching.<br>
```

Comment:
ResponeCachingStrategy 
spelling mistake.
I see I have another one a few lines down (implentation), can you fix that also?